js/cli/src/matches.ts

```diff
@@ -581,3 +572,22 @@ function setLogLevel(value, prev) {
 }
 
 program.parse(process.argv);
+
+async function initMatchesProgram(
```

Comment:
There is already a function for this in sol-command. https://github.com/raindrops-protocol/raindrops/blob/81c0a8ef80a1cd7bd11393e3555d8da2253b8fb0/js/cli/src/item.ts#L26

Reply:
Below that line is also the preferred way to create the program object from the CLI.